src/buffer/channel.rs

```diff
@@ -24,13 +24,19 @@ pub fn view<'a>(
         .get_channel_messages(&state.server, &state.channel)
         .into_iter()
         .filter_map(|message| {
-            let nickname = message.nickname().unwrap_or_default();
+            let Some(user) = message.user() else {
+                return None
+            };
```

Comment:
You can just `message.user()?` throw since we are returning an Option

Reply:
Ah doh, of cause.